# file: exception.py

class NotFound(Exception):
    pass
